Source/Widgets/CesiumInspector/CesiumInspectorViewModel.js

```diff
@@ -104,247 +104,243 @@ define([
+         * @type {String}
+         * @default '1 of 1'
+         */
+        this.depthFrustumText = knockout.observable('1 of 1');
+
+        /**
+         * Gets the text on the general section expand button.  This property is computed.
          * @type {String}
          * @default '-'
          */
-        this.generalSwitchText = '-';
+        this.generalSwitchText = knockout.pureComputed(function() {
+            return that.generalVisible ? '-' : '+';
+        });
 
         /**
-         * Gets or sets if the text on the primitive section expand button.  This property is observable.
+         * Gets the text on the primitives section expand button.  This property is computed.
          * @type {String}
          * @default '+'
          */
-        this.primitivesSwitchText = '+';
+        this.primitivesSwitchText = knockout.pureComputed(function() {
+            return that.primitivesVisible ? '-' : '+';
+        });
 
         /**
-         * Gets or sets if the text on the terrain section expand button.  This property is observable.
+         * Gets the text on the terrain section expand button.  This property is computed.
          * @type {String}
          * @default '+'
          */
-        this.terrainSwitchText = '+';
-
-        knockout.track(this, ['filterTile', 'suspendUpdates', 'dropDownVisible', 'shaderCacheText', 'frustums',
-                              'frustumStatisticText', 'pickTileActive', 'pickPrimitiveActive', 'hasPickedPrimitive',
-                              'hasPickedTile', 'tileText', 'generalVisible', 'generalSwitchText',
-                              'primitivesVisible', 'primitivesSwitchText', 'terrainVisible', 'terrainSwitchText', 'depthFrustumText']);
+        this.terrainSwitchText = knockout.pureComputed(function() {
+            return that.terrainVisible ? '-' : '+';
+        });
 
         this._toggleDropDown = createCommand(function() {
             that.dropDownVisible = !that.dropDownVisible;
         });
 
         this._toggleGeneral = createCommand(function() {
             that.generalVisible = !that.generalVisible;
-            that.generalSwitchText = that.generalVisible ? '-' : '+';
         });
 
         this._togglePrimitives = createCommand(function() {
             that.primitivesVisible = !that.primitivesVisible;
-            that.primitivesSwitchText = that.primitivesVisible ? '-' : '+';
         });
 
         this._toggleTerrain = createCommand(function() {
             that.terrainVisible = !that.terrainVisible;
-            that.terrainSwitchText = that.terrainVisible ? '-' : '+';
         });
 
-        this._showFrustums = createCommand(function() {
-            if (that.frustums) {
-                that._scene.debugShowFrustums = true;
-            } else {
-                that._scene.debugShowFrustums = false;
-            }
-            return true;
+        this.frustums.subscribe(function(val) {
```

Comment:
Any time we do a `subscribe`, we need to do dispose of it when the widget is destroyed.  So do something like

`this._frustumsSubscription = this.frustums.subscribe(...)`

then in `CesiumInspectorViewModel.prototype.detroy` do `this._frustumsSubscription.dispose()`